pkg/proxydisc/error.go

```diff
@@ -0,0 +1,30 @@
+package proxydisc
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+)
+
+// HTTPError represents an HTTP error.
+type HTTPError struct {
+	Status int    `json:"http_status"` // HTTP Status.
```

Comment:
Is there a reason why struct and JSON names differ?

Reply:
No reason. I'll change it.